src/components/SigningModals/MultiScreenModal.js

```diff
@@ -0,0 +1,171 @@
+import React, { useCallback, useState } from 'react'
+import { Spring, Transition, animated } from 'react-spring'
+import PropTypes from 'prop-types'
+import {
+  GU,
+  Modal,
+  Viewport,
+  springs,
+  textStyle,
+  useViewport,
+} from '@aragon/ui'
+import { useSteps } from '../../hooks'
+
+const AnimatedDiv = animated.div
+
+function MultiScreenModal({ visible, screens, onClose }) {
+  const steps = screens.length
+  const { prev, step, next, direction } = useSteps(steps)
+  const [measuredHeight, setMeasuredHeight] = useState(false)
+  const [height, setHeight] = useState(null)
+  const [firstStart, setFirstStart] = useState(true)
+  const { below } = useViewport()
+
+  const smallMode = below('medium')
+  const { canClose = true, width = 680 } = screens[step]
+
+  const renderScreen = useCallback(
+    step => {
+      const currentScreen = screens[step]
+      const { title } = currentScreen
+
+      return (
+        <React.Fragment>
+          <h1
+            css={`
+              ${smallMode ? textStyle('title4') : textStyle('title3')};
+
+              line-height: 1.2;
```

Comment:
Can we achieve this with the margin below?

Reply:
This was to avoid excess whitespace above the node which doesn't look as tightly aligned inside the box. I think what I'll do is use a negative margin to achieve the same thing without modifying the `line-height`.